test/test_client.py

```diff
@@ -1837,22 +1837,22 @@ def test_handshake_08_invalid_aws_ec2(self):
         )
 
     def test_dict_hints(self):
-        c = rs_or_single_client()
+        c = self.client
```

Comment:
Since we're at it can we clean these tests up a bit more? The try/except is redundant since any exception will case the test to fail anyway. I'd suggest:
```python
def test_dict_hints(self):
    self.db.t.find(hint={"x": 1})
...
```